force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +336,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        list <String> matchFound = new List<String>();
+        Integer wordIndex = 1;
+        
         // Loop through the list of words
-            // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        for (String w : words){
```

Comment:
Try to have meaningful names for variables instead of single letters. 
ex. 'wordsThatIdLikeTofind' is better than 'w'